cpu/cc2538/radio/cc2538_rf_netdev.c

```diff
@@ -361,8 +361,6 @@ static int _init(netdev_t *netdev)
     netdev_ieee802154_reset(&dev->netdev);
 
     /* Initialise netdev_ieee802154_t struct */
-    netdev_ieee802154_set(&dev->netdev, NETOPT_NID,
-                          &pan, sizeof(pan));
```

Comment:
The [line above](https://github.com/RIOT-OS/RIOT/pull/10384/files#diff-b8d088b7424905055347a5d1f7bd781cR356) isn't needed anymore ;-)

Reply:
Removed!